### ISSUE TITLE
Plug-in Installation 

### ISSUE THREAD

**Issue by jbacksch** [bartnv/twofactor_webauthn#5]
I've copied the Webauthn plug-in into the plug-in directory of roundcoube, but the plug-in does not occur.

**Comment by Technikte** [comment]
@jbacksch Have you done this ?

> For any later activation or deactivation of a plugin, you can add/remove the plugin name to/from the $config['plugins'] option in your local Roundcube config file config/config.inc.php.

taken from https://plugins.roundcube.net/#/.

An example config.inc.php..
```
$config['plugins'] = array(
	'attachment_reminder',
	'enigma',
	'show_folder_size',
	'twofactor_webauthn',
);
```


Please check if you fullfill all Requirements that are listed in the readme here on github, most important the https:// connection to your roundcube installation.

**Comment by exstntlmsnthrp** [comment]
I seem to be having a similar issue to here. I have verified and meet the requirements listed in the readme file. I have also followed the instructions at [https://plugins.roundcube.net/#/](https://plugins.roundcube.net/#/), including I have also added it to my composer.json file.

My problem is that both the instructions listed above and composer require me to have a ``version constraint`` in the composer.json file and I cannot find one available here on github or at [packagist](https://packagist.org/packages/mmvi/twofactor_webauthn). Composer does not accept ``~dev-master``.

Thank you for your assistance.
